### PR TITLE
docs(install): add x-cmd method to install oha

### PR DESCRIPTION
**Pull request by curry798**
Adding a new installation method to the README.
from [issue](https://github.com/hatoo/oha/issues/601).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,6 +71,14 @@ bun run pgo.js
...
 And the binary will be available at `target/[target-triple]/pgo/oha`.
 
+## X-CMD
+
+You can install with [x-cmd](https://www.x-cmd.com).
+
+```sh
+x env use oha
+```
+
```
Comment (hatoo) [inline README.md:81]: Can you move it to the `Install` section?

**Review by hatoo — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (curry798) [reply README.md]: > Can you move it to the `Installation` section?

ok, I moved it before the Containerized section, please take another look.

Comment (curry798) [reply README.md]: > Can you move it to the `Installation` section?你能把它移到`Installation`部分吗？

Or should I put it before `On Arch Linux`, alongside "cargo"?

**Comment by hatoo** [comment]
OK, thank you!

**Pull request merged**